todo_app/data/session_items.py

```diff
@@ -171,3 +204,56 @@ def markcomplete(itemid):
       headers=headers,
       params=query
    )
+
+
+#######Create a new board ######
+def create_trello_board():
+   url = "https://api.trello.com/1/boards/"
+
+
+
+   query = {
+      'key': os.getenv('KEY'),
+      'token': os.getenv('TOKEN'),
+      'name': 'Assignments'
+   }
+
+
+   response = requests.request(
+      "POST",
+      url,
+      params=query
+   )
+   getresponse = response.text
+   
+   getresponse1 = json.loads(getresponse)
```

Comment:
We can simplify this with:
```python
getresponse = response.json()
```
To skip reading it as text and then separately parsing to json

Reply:
Done